sdk/keyvault/test-resources.json

```diff
@@ -37,7 +37,7 @@
         },
         "hsmLocation": {
             "type": "string",
-            "defaultValue": "westus",
+            "defaultValue": "canadacentral",
```

Comment:
We will still pass in `westus2` as the default in the deployment scripts unless an override is specified in `tests.yml`